mscore/editstaff.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>760</width>
-    <height>600</height>
+    <width>797</width>
+    <height>757</height>
```

Comment:
Can't we avoid this change? Just not commit this chunk?

Reply:
@Jojo-Schmitz 
I was forced to increase the height (not sure when the width changed) because there wasn't enough space for the spinboxes. The thing is that as I've said in the Telegram chat, I have 2 conflicting pull requests with UI changes. No matter which one gets merged first I'll be forced to make changes in the UI.